pages/dkp/kaptain/2.0.0/install/air-gapped-2.1/index.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: layout.pug
+navigationTitle: Add Kaptain air-gapped for DKP 2.1
+title: Add Kaptain to your DKP Catalog applications on air-gapped environments for DKP 2.1
+menuWeight: 10
+excerpt: Add Kaptain to your DKP Catalog applications on air-gapped environments for DKP 2.1
+beta: false
+enterprise: false
+---
+
+<p class="message--warning"><strong>WARNING: </strong>
+You can deploy Kaptain to a cluster in a selected workspace. If you do not intend to deploy Kaptain to a certain cluster, you must switch the workspace you are deploying to or move that cluster to another workspace.
+</p>
+
+## DKP 2.1 air-gapped installation
+
+Refer to [DKP install instructions][dkp_install], if you want to deploy Kaptain in a networked environment or to [DKP 2.2 air-gapped instructions][2.2_air] if you are deploying in DKP 2.2.
+
+Kaptain supports installation on an air-gapped (a.k.a. offline or private) DKP managed cluster. Before installing Kaptain, please follow the [air-gapped installation guide][konvoy-air-gap] to set up the air-gapped DKP managed cluster. The cluster admin is responsible for configuring the DKP cluster correctly and ensuring container images have been pre-loaded to the private registry before installing Kaptain.
+
+## Prerequisites
+
+-   A DKP cluster with the following Platform applications enabled:
+
+    - Istio
+    - Knative (optional, if KServe is configured to work in `RawDeployment` mode)
+
+-   [`kubectl`][kubectl] on your installation machine
+
+-   For customers deploying in a multi-cluster environment (Enterprise): Ensure you have configured [Kaptain to authenticate with a Management Cluster][dex].
+
+-   Ensure the following applications are enabled in Kommander:
+
+    1. Use the existing Kommander configuration file, or initialize the default one:
+
+		```bash
+		dkp install kommander --init > kommander-config.yaml
+		```
+
+    1. Ensure the following applications are enabled in the config:
+
+		```yaml
+		apiVersion: config.kommander.mesosphere.io/v1alpha1
+		kind: Installation
+		apps:
+		  ...
+		  dex:
+		  dex-k8s-authenticator:
+		  kube-prometheus-stack:
+		  istio:
+		  knative:
+		  minio-operator:
+		  traefik:
+		  nvidia:  # to enable GPU support
+		  ...
+		```
+
+    1. Apply the new configuration to Kommander:
+
+		```bash
+		dkp install kommander --installer-config kommander-config.yaml
+		```
+
+<!-- TODO: Reference doc for DKP, as commands are different depending on the installation and delete only this step or the whole "Ensure the following applications are enabled in Kommander:" part?-->
```

Comment:
@lukeogg @alembiewski 
Should we delete lines 32 to 64 and refer to this doc: https://docs.d2iq.com/dkp/kommander/2.1/install/configuration/ (I am not sure which document is the right one to reference here), making a wild guess: 
```suggestion
[Configure a Kommander installation](../../../../kommander/2.1/install/configuration) to include `dex`, `dex-k8s-authenticator`, `kube-prometheus-stack`, `istio`, `minio-operator`, `traefik`, `nvidia` and `knative` (Knative is optional if KServe is configured to work in `RawDeployment` mode).
```